### PR TITLE
Add a scaffold for C#

### PR DESCRIPTION
**Pull request by poulinjulien**
It still need some work regarding the Docker integration, but the main parts are here...

### CODE REVIEW

File: scaffolds/csharp/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+FROM microsoft/dotnet:latest
```
Comment (jbruggem) [inline scaffolds/csharp/Dockerfile]: specify version !

File: scaffolds/csharp/README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+There are several options. The easiest way is to work with a docker image ready
+with all the tools you need:
+
+```Bash
+docker build -t csharp-code-retreat .
+```
+
+The only downside is that you need to run the .NET CLI through Docker. 
+//To make it easy, we wrote a script `gradle.sh` wrapping the `docker` command and acting as the .NET CLI
```
Comment (jbruggem) [inline scaffolds/csharp/README.md]: gradle.sh ?

File: scaffolds/csharp/README.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+## Run the tests
+
+```Bash
+dotnet test
```
Comment (jbruggem) [inline scaffolds/csharp/README.md]: isn't this supposed to use docker ?

**Comment by jbruggem** [comment]
Are xproj and sln files necessary to run this code ? Are those not IDE-specific ?

File: scaffolds/csharp/README.md (hunk 2 above)
Comment (poulinjulien) [inline scaffolds/csharp/README.md]: Yeah, I copy / pasted from the Java scaffold, but it's not done yet... I was able to run it in docker, but I don't know how to automate it (using a script, I guess).

File: scaffolds/csharp/README.md (hunk 3 above)
Comment (poulinjulien) [inline scaffolds/csharp/README.md]: Yes

**Comment by poulinjulien** [comment]
@jbruggem I think they are still required by the build system.

File: scaffolds/csharp/README.md (hunk 2 above)
Comment (jbruggem) [inline scaffolds/csharp/README.md]: Want some help with that ? I'd be happy to

**Comment by jbruggem** [comment]
@poulinjulien if you're not certain then let's try so that we can know for sure :)

File: scaffolds/csharp/README.md (hunk 2 above)
Comment (poulinjulien) [inline scaffolds/csharp/README.md]: Yes, please :-)

File: scaffolds/csharp/README.md [hunk 4]
```diff
@@ -9,8 +9,11 @@ with all the tools you need:
 docker build -t csharp-code-retreat .
 ```
 
-The only downside is that you need to run the .NET CLI through Docker. 
-//To make it easy, we wrote a script `gradle.sh` wrapping the `docker` command and acting as the .NET CLI
+The only downside is that you need to run the .NET CLI through Docker.
+
+```Bash
+docker run -it csharp-code-retreat
```
Comment (toch) [inline scaffolds/csharp/README.md]: according your script the docker image name doesn't correspond. does it refer to something else?

**Comment by toch** [comment]
Considering the #12 has been merged, is this PR still valid? @jbruggem @poulinjulien

**Comment by jbruggem** [comment]
I guess not ! @poulinjulien are you ok if we close ?

**Pull request closed without merging**